web_modules/Navigation/index.js

```diff
@@ -48,6 +48,14 @@ export default class Navigation extends Component {
               { "API" }
             </a>
           </li>
+          <li className={ styles.item }>
+            <a
+              className={ styles.itemLink }
+              href="https://github.com/postcss/brand"
+            >
+              { "Logos/Graphics" }
```

Comment:
I think this should change to **Brand**.

```suggestion
              { "Brand" }
```